explanation/intro-to/virtualisation.md

```diff
@@ -1,24 +1,28 @@
-(introduction-to-virtualisation)=
-# Introduction to virtualisation
+(introduction-to-virtualization)=
+# Introduction to virtualization
 
-Virtualisation is a technology that allows you to create safe, isolated environments on your server. For developers, this often means creating self-contained sandboxes for development and testing that cannot negatively affect the host machine. For systems administrators, it allows resources to be scaled to meet changing demand, giving greater control and flexibility in managing infrastructure.
+Virtualization is a technology that allows you to create safe, isolated environments on your server. Developers can use virtualization to create self-contained sandboxes for development and testing that won't negatively affect the host machine. System administrators can use virtualization to effectively scale network resources to meet changing demand.
```

Comment:
```suggestion
Virtualization is a technology that allows you to create safe, isolated environments on your server. Developers can use virtualization to create self-contained sandboxes for development and testing that cannot negatively affect the host machine. System administrators can use virtualization to scale network resources to meet changing demand, giving greater control and flexibility in managing infrastructure.
```

Reply:
Again, we want to be careful with wording choices to make sure they don't change the meaning of what's being communicated. Changing "cannot" to "won't" implies that a sandbox could-but-doesn't affect the host machine, but they are designed so that they quite literally cannot. I'd also like to keep the point about control and flexibility - it's the whole reason why virtualization is so useful to sysadmins and should be mentioned :)